presto-accumulo/pom.xml

```diff
@@ -363,6 +367,12 @@
             <artifactId>jackson-databind</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>log4j</groupId>
+            <artifactId>log4j</artifactId>
+            <version>1.2.17</version>
```

Comment:
Wouldn't it be weird to put `log4j` in the `presto-root` pom, since we want to get rid of it but are kind of working around the issue?